lambda-functions/event-receiver/app/makeregister_event_handler.go

```diff
@@ -79,3 +86,32 @@ func handleUsers(ctx context.Context, dynamoClient DynamodbClient, actor Actor)
 
 	return nil
 }
+
+func handleLpas(ctx context.Context, dynamoClient DynamodbClient, actor Actor, userId string, LpaId string) error {
+	var existingLPA map[string]types.AttributeValue
+
+	err := dynamoClient.GetByLpaIDAndUserID(ctx, LpaId, userId, &existingLPA)
+	if err == nil && existingLPA != nil {
+		return nil
+	}
```

Comment:
This logic took me a while to parse. Since I don't think you're doing anything with the existing value, I think it'd be easier if `GetByLpaIDAndUserID` returned a boolean of whether the LPA was found.
```suggestion
	exists, err := dynamoClient.GetByLpaIDAndUserID(ctx, LpaId, userId)
	if err == nil && exists {
		return nil
	}
```
At which point you could also rename it to something like `ExistsLpaIDAndUserID`

Reply:
Oh, also, I think `existingLpa` will never be `nil`. If it's not populated, it will be a zero-valued struct instead.